### ISSUE TITLE
[feature]: Add wireframes to the project

### ISSUE THREAD

**Issue by gbowne1** [gbowne1/reactsocialnetwork#188]
### Is your feature request related to a problem? Please describe.

Some have asked for a Figma or Wireframes to help them work on this project.

Add some wireframes that would get us up to where we are currently with our design.

This may improve contributions to our project.

### Describe the solution you would like

Add some wireframes from somewhere like draw.io 

### Describe the alternatives you have tried

We considered Figma designs to help ease developer.

**Comment by minnayu** [comment]
Hello, I would love to contribute to this issue, but I'm encountering an issue when starting the server. Do you have any solutions? 

<img width="1096" alt="Screen Shot 2023-04-25 at 12 47 30 AM" src="https://user-images.githubusercontent.com/69655024/234209651-ac1ad803-9a17-44b5-8f23-018f5c6b45c2.png">

**Comment by gbowne1** [comment]
@minnayu 

I have not tried it with any nodejs version beyond 18.  I noticed you're on 19.  

We have been experiencing sone problems with packages.

We are waiting on a PR review to add package updates.

We recently also added eslint and prettier configurations to help with keeping our code clean and pretty.

I am certainly not a Babel, Webpack, ESLint or Prettier expert.  Hopefully fixes will come soon.

**Comment by gbowne1** [comment]
This is a duplicate issue. Closing this as a duplicate of wireframe issue.
